packages/dds/tree/src/test/shared-tree/fuzz/operationTypes.ts

```diff
@@ -116,7 +146,7 @@ export interface CrossFieldMove extends Move {
 	 * The field to move the content to.
 	 * May be the same as the source field.
 	 */
-	dstField: FieldDownPath;
+	dstField: DownPath | undefined;
```

Comment:
```suggestion
	dstParent: DownPath | undefined;
```
Since this isn't describing the path to a field anymore.